src/etc/inc/interfaces.inc

```diff
@@ -4004,9 +4016,25 @@ function interface_dhcpv6_configure($interface = "wan", $wancfg) {
 	$debugOption = isset($wancfg['dhcp6debug']) ? "-D" : "-d";
 	$rtsoldscript .= "/usr/local/sbin/dhcp6c {$debugOption} -c {$g['varetc_path']}/dhcp6c_{$interface}.conf -p {$g['varrun_path']}/dhcp6c_{$wanif}.pid {$wanif}\n";
 	$rtsoldscript .= "/usr/bin/logger -t rtsold \"Starting dhcp6 client for interface {$interface}({$wanif})\"\n";
+	/* non ipoe Process */
+ 	if (!isset($wancfg['dhcp6withoutra'])){	
+		$rtsoldscript .= "if [ -f {$g['varrun_path']}/dhcp6c_{$wanif}.pid ]; then\n";
+		$rtsoldscript .= "\t/bin/pkill -F {$g['varrun_path']}/dhcp6c_{$wanif}.pid\n";
+		$rtsoldscript .= "\t/bin/sleep 1\n";
+		$rtsoldscript .= "fi\n";
+	}
+	else
+	{
+		$rtsoldscript .= "\t/bin/sleep 1\n";
+	}
+	$debugOption = isset($wancfg['dhcp6debug']) ? "-D" : "-d";
+	if (!isset($wancfg['dhcp6withoutra'])){
```

Comment:
More little style stuff
s/){/) {
There are a couple of places that need the space between ")" and "{".